lib/msal-common/src/utils/Constants.ts

```diff
@@ -17,6 +17,9 @@ export const Constants = {
     DSTS: "dstsv2",
     // Default AAD Instance Discovery Endpoint
     AAD_INSTANCE_DISCOVERY_ENDPT: "https://login.microsoftonline.com/common/discovery/instance?api-version=1.1&authorization_endpoint=",
+    // CIAM URL
+    CIAM_AUTH_URL: ".ciamlogin.com",
+    CIAM_ENDPOINT: ".onmicrosoft.com",
```

Comment:
> CIAM_ENDPOINT: ".onmicrosoft.com",

super nit: maybe something like `AAD_TENANT_DOMAIN_SUFFIX` would be more accurate

Reply:
Done